### PR TITLE
fix!(http_server source): only single query param when namespace is enabled

### PR DESCRIPTION
**Pull request by Zettroke**
Http server has different behavior when handling query params if namespace is enable or not.
When namespace is enable we getting only last query parameter out of specified parameters list. 
Without namespace we are getting all of specified parameters as key value pairs.

Pull request aligns this behavior by making `%http_server.query_parameters` an object with query params.

### CODE REVIEW

**Review by jszwedko — COMMENTED** [review]
Nice catch, thanks @Zettroke . Could we:

- Add a test for this? I'd be ok with a test directly in `query.rs` here if that is simplest
- Add a changelog fragment? See https://github.com/vectordotdev/vector/blob/master/changelog.d/README.md

**Comment by Zettroke** [comment]
@jszwedko done

File: changelog.d/http-server-query-params-with-namespace.fix.md [hunk 1]
```diff
@@ -0,0 +1 @@
+http_server source now properly writes all the specified query parameters to `%http_server.query_parameters`
```
Comment (jszwedko) [inline changelog.d/http-server-query-params-with-namespace.fix.md:1]:
```suggestion
The `http_server` source now properly writes all the specified query parameters to `%http_server.query_parameters` when log namespacing is enabled.

authors: Zettroke
```

**Review by jszwedko — COMMENTED** [review]
One note about the changelog, otherwise 👍

**Review by jszwedko — APPROVED** [review]
Thanks for this contribution @Zettroke !

**Review by jszwedko — APPROVED** [review]

**Pull request merged**